lib/rules/require-toggle-inside-transition.js

```diff
@@ -61,12 +104,18 @@ module.exports = {
         element.name !== 'slot' &&
         !utils.hasDirective(element, 'if') &&
         !utils.hasDirective(element, 'show') &&
-        !utils.hasDirective(element, 'bind', 'key')
+        !utils.hasDirective(element, 'bind', 'key') &&
+        !additionalDirectives.some((directive) =>
+          utils.hasDirective(element, directive)
+        )
```

Comment:
With the above suggestion, this could be simplified to:

```js
      if (
        element.name !== 'slot' &&
        !allowedDirectives.some((directive) =>
          utils.hasDirective(element, directive)
        ) &&
        !utils.hasDirective(element, 'bind', 'key')
      )
```